OpenSubtitlesHandler/Util.cs

```diff
@@ -0,0 +1,152 @@
+        {
+            Util._version = version;
+        }
+
+        /// <summary>
+        /// Compute movie hash
+        /// </summary>
+        /// <returns>The hash as Hexadecimal string</returns>
+        public static string ComputeHash(Stream stream)
+        {
+            var hash = ComputeMovieHash(stream);
+            return Convert.ToHexString(hash).ToLowerInvariant();
+        }
+
+        /// <summary>
+        /// Serialize object into JSON
+        /// </summary>
+        /// <returns>JSON string of the object</returns>
+        public static string Serialize(object o)
+        {
+            return JsonSerializer.Serialize(o);
+        }
+
+        /// <summary>
+        /// Deserialize object from JSON
+        /// </summary>
+        /// <returns>Deserialized object</returns>
+        public static T Deserialize<T>(string str)
+        {
+            return JsonSerializer.Deserialize<T>(str, SerializerOpts);
+        }
+
+        /// <summary>
+        /// Compute hash of specified movie stream
+        /// </summary>
+        /// <returns>Hash of the movie</returns>
+        private static byte[] ComputeMovieHash(Stream input)
+        {
+            using (input)
+            {
+                var streamSize = input.Length;
+                var lHash = streamSize;
+
+                long i = 0;
+                byte[] buffer = new byte[sizeof(long)];
+
+                while (i < 65536 / sizeof(long) && (input.Read(buffer, 0, sizeof(long)) > 0))
+                {
+                    i++;
+                    lHash += BitConverter.ToInt64(buffer, 0);
+                }
+
+                input.Position = Math.Max(0, streamSize - 65536);
+                i = 0;
+
+                while (i < 65536 / sizeof(long) && (input.Read(buffer, 0, sizeof(long)) > 0))
+                {
+                    i++;
+                    lHash += BitConverter.ToInt64(buffer, 0);
+                }
+
+                byte[] result = BitConverter.GetBytes(lHash);
+                Array.Reverse(result);
+
+                return result;
+            }
+        }
+
+        internal static async Task<(string, Dictionary<string, string>, HttpStatusCode)> SendRequestAsync(string url, HttpMethod method, object body, Dictionary<string, string> headers, CancellationToken cancellationToken)
+        {
+            if (!HttpClient.DefaultRequestHeaders.Contains("User-Agent"))
+            {
+                if (string.IsNullOrWhiteSpace(_version))
+                {
+                    throw new Exception("Missing plugin version");
+                }
+
+                var ua = $"Jellyfin-Plugin-OpenSubtitles/{_version}";
+
+                HttpClient.DefaultRequestHeaders.Add("User-Agent", ua);
+            }
+
+            HttpContent content = null;
+            if (method != HttpMethod.Get && body != null)
+            {
+                content = new StringContent(Util.Serialize(body), Encoding.UTF8, "application/json");
```

Comment:
`MediaTypeNames.Application.Json` instead of `"application/json"`

Reply:
Also done in 65763b6e4766b8c5c8362bd63cd94698a313c6bb